<launch>
    <!-- parameters -->
    <arg name="iris_config_path" default = "$(find iris)/../config/hongo-vinsmono.yaml" /> 
    <arg name="pcd_path" default = "$(find iris)/../../../hongo.pcd" /> 
    
    <!-- iris_node -->
    <node name="iris_node" pkg="iris" type="iris_node" output="screen">
        <param name="iris_config_path" type="string" value="$(arg iris_config_path)" />
        <param name="pcd_path" type="string" value="$(arg pcd_path)" />
    </node>

    <!-- vinsmono_bridge_node -->
     <node name="vinsmono_bridge_node" pkg="vinsmono_bridge" type="vinsmono_bridge_node" output="screen">
    </node>

    <!-- rviz -->
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find iris)/../config/iris-vinsmono.rviz" />
</launch>
